qcodes/instrument_drivers/tektronix/Keithley_3706A_Matrix_channels.py

```diff
@@ -0,0 +1,600 @@
+                           docstring='Enables or disables LAN connection.',
+                           val_mapping=create_on_off_val_mapping(on_val='true',
+                                                                 off_val='false'
+                                                                 ))
+
+        self.connect_message()
+
+    def _reset_channel(self, val: str) -> None:
+        if not self._validator(val):
+            raise InvalidValue(f'{val} is not a valid specifier. '
+                               'The specifier should be channels, channel '
+                               'ranges, slots, backplane relays or "allslots".')
+        self.write(f"channel.reset('{val}')")
+
+    def _open_channel(self, val: str) -> None:
+        if not self._validator(val):
+            raise InvalidValue(f'{val} is not a valid specifier. '
+                               'The specifier should be channels, channel '
+                               'ranges, slots, or "allslots".')
+        self.write(f"channel.open('{val}')")
+
+    def _close_channel(self, val: str) -> None:
+        slots = ['allslots', *self._get_slot_name()]
+        if val in slots:
+            raise InvalidValue("Slots cannot be closed all together.")
+        if not self._validator(val):
+            raise InvalidValue(f'{val} is not a valid specifier. '
+                               'The specifier should be channels or channel '
+                               'ranges and associated backplane relays.')
+        self.write(f"channel.close('{val}')")
+
+    def _set_exclusive_close(self, val: str) -> None:
+        slots = ['allslots', *self._get_slot_name()]
+        if val in slots:
+            raise InvalidValue("Slots cannot be exclusively closed.")
+        if val == "":
+            raise InvalidValue('An empty string may cause all channels and '
+                               'associated backplane relays to open. Use '
+                               '"open_channel" parameter instead.')
+        if not self._validator(val):
+            raise InvalidValue(f'{val} is not a valid specifier. '
+                               'The specifier should be channels or channel '
+                               'ranges and associated backplane relays.')
+        self.write(f"channel.exclusiveclose('{val}')")
+
+    def _set_exclusive_slot_close(self, val: str) -> None:
+        slots = ['allslots', *self._get_slot_name()]
+        if val in slots:
+            raise InvalidValue("Slots cannot be exclusively closed.")
+        if val == "":
+            raise InvalidValue('Argument cannot be an empty string.')
+        if not self._validator(val):
+            raise InvalidValue(f'{val} is not a valid specifier. '
+                               'The specifier should be channels or channel '
+                               'ranges and associated backplane relays.')
+        self.write(f"channel.exclusiveslotclose('{val}')")
+
```

Comment:
Another potentially useful convenience function would be something like connect all rows in a slot to a given column or connect all columns in a slot to a given row.